presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/source/BenchmarkSuiteConfig.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.source;
+
+import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.NotNull;
+
+public class BenchmarkSuiteConfig
+{
+    private String database;
+    private String suite;
+    private String suitesTableName = "benchmark_suites";
+    private String queriesTableName = "benchmark_queries";
+
+    @NotNull
+    public String getDatabase()
+    {
+        return database;
+    }
+
+    @Config("database")
+    public BenchmarkSuiteConfig setDatabase(String database)
+    {
+        this.database = database;
+        return this;
+    }
```

Comment:
Move database to a separate config. `database` is only specific to mysql suite supplier, and is not applicable to xdb suite supplier. The rest of the 3 configs (suite, suite table, query table), or applicable to db suite supplier (for both mysql and xdb) in general.